readthedocs/doc_builder/python_environments.py

```diff
@@ -174,6 +176,7 @@ def is_obsolete(self):
 
         env_python = environment_conf.get('python', {})
         env_build = environment_conf.get('build', {})
+        env_vars_hash = environment_conf.get('env_vars_hash', None)
```

Comment:
Defaulting to `None` will cause to every build before this change to wipe. Not really sure if that will be a problem, but we could default to the hash of an empty string here to prevent that.

Reply:
Yeahhh.. that is better.
Should I generate it every time or should I copy the value from the web for the sha256 hash of empty string?

Reply:
Defaulting to `None` is fine.

The internal method should return `None` as well if there are no Environment Variables.

Reply:
The internal method returns the hash of the empty string

Reply:
@humitos @stsewd 
What should we do here?